rareapi/views/Posts.py

```diff
@@ -93,7 +93,16 @@ def retrieve(self, request, pk=None):
             return Response(serializer.data)
         except Exception as ex:
             return HttpResponseServerError(ex)
-            
+
+    def destroy(self, request, pk=None):
+        try:
+            post = Posts.objects.get(pk=pk)
+            post.delete()
+            return Response({}, status=status.HTTP_204_NO_CONTENT)
+        except Posts.DoesNotExist as ex:
+            return Response({}, status=status.HTTP_404_NOT_FOUND)
```

Comment:
We should still return the response from the exception: `return Response({'message': ex.args[0]}, status=status.HTTP_404_NOT_FOUND)` , otherwise the client will just get an empty object back.